resources/public/api/conf/1.0/common/errors.yaml

```
@@ -58,7 +58,7 @@ components:
 
     formatLatencyIndicator:
       description: |
-        The format of a latency indicator fields must be "Annual" or "Quarterly".
+        The format of a latency indicator fields must be "A" or "Q".
```

Here and in the tech spec, can you explain what A and Q mean?

e.g. 
```suggestion
        The format of a latency indicator fields must be "A" or "Q" (Annual or Quarterly, respectively).
```